src/backend/marsha/core/xapi.py

```diff
@@ -86,8 +87,9 @@ def _enrich_statement(self, statement, video, lti_user):
         if "id" not in statement:
             statement["id"] = str(uuid.uuid4())
 
-        statement["timestamp"] = statement["timestamp"].isoformat()
-
+        statement["timestamp"] = (
+            datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc).isoformat()
```

Comment:
You should use Django's localized utils:
```
from django.utils import timezone
timezone.now() 
```

Reply:
Done